README.md

```diff
@@ -13,80 +13,73 @@
 <!-- <img src="TODO.jpg" width="200" height="200" /> -->
 
 
-Overview
-----
-Project Concord is a highly scalable and energy-efficient distributed trust infrastructure for consensus and smart contract execution.
+## Overview
 
-concord-bft
-----
+**Concord-bft** is a generic state machine replication library that can handle malicious (byzantine) replicas.
 
-
-The core engine of project Concord is concord-bft, a generic state machine replication library that can handle malicious (byzantine) replicas.
-This library is designed to be used as a core building block for replicated distributed data stores, and is especially suited to serve as the basis of permissioned Blockchain systems.
 Its implementation is based on the algorithm described in the paper [SBFT: a Scalable Decentralized Trust Infrastructure for
 Blockchains](https://arxiv.org/pdf/1804.01626.pdf).
 
-Releases
-----
+It is designed to be used as a core building block for replicated distributed data stores, and is especially suited to serve as the basis of permissioned Blockchain systems.
+
+For a real-life integration example, please take a look at [Project Concord](https://github.com/vmware/concord), a highly scalable and energy-efficient distributed trust infrastructure for consensus and smart contract execution.
+
+
+## Releases
 
 We abide by [semantic versioning](https://semver.org/). Public APIs can change
 at any time until we reach version 1.0. We will, however, bump the minor version
 when a backwards incompatible change is made.
 
  [v0.5](https://github.com/vmware/concord-bft/releases/tag/v0.5)
 
-Contributing
-----
-
-The concord-bft project team welcomes contributions from the community. If you wish to contribute code and you have not
-signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any
-questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq). For more detailed information,
-refer to [CONTRIBUTING.md](CONTRIBUTING.md).
-
-Community
-----
-
-[Concord-BFT Slack](https://concordbft.slack.com/).
-
-Get Slack invitation via this [link](https://join.slack.com/t/concordbft/shared_invite/enQtNTAyMzg0MTk0NzU0LWViMjA4ODQ4OTcxYmVkMWU4NGYyNTZmZDQ0ZTIzZTNiZmIyZmI5Mjg0NDZjNjBkYmI2Yjg1Mzg1NGUyNjQ0OWQ) or send request to <concordbft@gmail.com>.
-
+## Build (Ubuntu Linux 18.04)
 
+We use Conan Package Manager to install all concord-bft dependencies.
+Dependencies that currently are not supported by the conan center, have custom conan installer in concord-bft/.conan
```

Comment:
grammar/typos:
- "...that are currently not supported..."
- "...have a custom..."

Reply:
that's not my text, the diff falsely reports as new.

This needs further iterations anyways, needs to become even simpler.